src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseView.java

```diff
@@ -22,11 +22,11 @@ public void setChannels(String[] channels) {
         this.channels = channels;
     }
 
-    public CaseViewTrigger[] getTriggers() {
```

Comment:
we should rename the method names here, otherwise it's confusing

Reply:
done